lib/widgets/message_list.dart

```diff
@@ -339,8 +339,9 @@ class MessageListAppBarTitle extends StatelessWidget {
     return Row(
       mainAxisSize: MainAxisSize.min,
       children: [
-        Flexible(child: Text(topic.displayName, style: const TextStyle(
+        Flexible(child: Text(topic.displayName ?? store.realmEmptyTopicDisplayName, style: TextStyle(
```

Comment:
Let's have a test for this part too. Should be quick, I think; can go in the "app bar" group.